src/lib/FlightTasks/tasks/ManualAltitude/FlightTaskManualAltitude.cpp

```diff
@@ -345,10 +345,17 @@ void FlightTaskManualAltitude::_updateSetpoints()
 	_respectGroundSlowdown();
 }
 
+bool FlightTaskManualAltitude::_checkTakeoff()
+{
+	// stick is is deflected up
+	return _sticks(2) > 0.65f;
```

Comment:
in this case the flag `want_to_takeoff` is always true?

Reply:
Yes, but it gets ignored when you're already flying.